bin/install-dependencies.sh

```diff
@@ -9,7 +9,7 @@ ckan_dcat_fork='alphagov'
 ckan_dcat_sha='be3b809fa7431c8d81508c01853e81ce6a5dfd84'
 
 ckan_spatial_fork='alphagov'
-ckan_spatial_sha='3f423de1e9e4e6975725712626d904887779b408'
+ckan_spatial_sha='2a21094b7e63ee48b8beb1229997827b66a62fe4'
```

Comment:
Should it be -

```suggestion
ckan_spatial_sha='433c7f2a4ba0b05871fe8bbc9b0518aa594f392d'
```

as grabbing the commit from `main-2.9` branch

Reply:
Lol, caught myself out with branches again. Yes, that's what it should be. I'll commit the suggestion.